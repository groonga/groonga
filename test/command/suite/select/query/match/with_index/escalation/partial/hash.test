table_create Users TABLE_NO_KEY
column_create Users name COLUMN_SCALAR ShortText
table_create Terms TABLE_HASH_KEY ShortText --default_tokenizer TokenDelimit
column_create Terms index COLUMN_INDEX Users name
load --table Users
[
{"name": "Alice"},
{"name": "Bob"}
]
select Users --filter 'name @ "Alice "'
