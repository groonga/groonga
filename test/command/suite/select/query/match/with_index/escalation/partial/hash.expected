table_create Users TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Users name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_HASH_KEY ShortText --default_tokenizer TokenDelimit
[[0,0.0,0.0],true]
column_create Terms index COLUMN_INDEX Users name
[[0,0.0,0.0],true]
load --table Users
[
{"name": "Alice"},
{"name": "Bob"}
]
[[0,0.0,0.0],2]
select Users --filter 'name @ "Alice "'
[[0,0.0,0.0],[[[0],[["_id","UInt32"],["name","ShortText"]]]]]
